src/components/dev-hub/layout.js

```diff
@@ -90,9 +90,16 @@ export default ({ children, includeCanonical = true }) => {
             <GlobalWrapper>
                 <Helmet htmlAttributes={{ lang: 'en' }}>
                     <meta name="robots" content="index" />
+                    <link
+                        type="application/opensearchdescription+xml"
+                        rel="search"
+                        href="https://developer.mongodb.com/opensearch.xml"
+                        title="Search MongoDB Developer Hub"
+                    />
                     <link
                         rel="shortcut icon"
                         href="https://www.mongodb.com/assets/images/global/favicon.ico"
+                        title="Search MongoDB Developer Hub"
```

Comment:
```suggestion
```
We should be able to omit `title` from this tag. It was the above link that should have it (and it is there now!)